plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMinioOrcConnectorTest.java

```diff
@@ -135,6 +135,25 @@ private void testReadSingleIntegerColumnOrcFile(String orcFileResourceName, int
         }
     }
 
+    @Test
+    public void testTimeType()
+            throws Exception
+    {
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_time", "(\"_col0\") AS VALUES " +
+                "(TIME '10:03:34.123000'), (NULL), (NULL), (NULL), (TIME '10:03:34.123000'), (TIME '10:03:34.123000')")) {
+            String orcFilePath = (String) computeScalar(format("SELECT DISTINCT file_path FROM \"%s$files\"", table.getName()));
+            try (OutputStream outputStream = fileSystem.newOutputFile(Location.of(orcFilePath)).createOrOverwrite()) {
+                Files.copy(new File(getResource("single-time-column.orc").toURI()).toPath(), outputStream);
```

Comment:
Does the file we're replacing and the `single-time-column.orc` have same length?